src/java/org/apache/cassandra/locator/ReplicaPlans.java

```diff
@@ -38,6 +40,7 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.Multimap;
 import com.google.common.collect.Sets;
+import org.apache.cassandra.config.CassandraRelevantProperties;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
```

Comment:
nit:  `org.apache.cassandra.` come last (but before statics)
```suggestion
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import org.apache.cassandra.config.CassandraRelevantProperties;
```